keyboards/feker/ik75/keymaps/bkzshen/keymap.c

```diff
@@ -195,8 +195,8 @@ bool rgb_matrix_indicators_user(void) {
     rgb_matrix_set_color(104, 0, 0, 0);
 
     uint8_t red = host_keyboard_led_state().caps_lock ? 255 : 0;
-    uint8_t green = host_keyboard_led_state().scroll_lock ? 255 : 0;
-    uint8_t blue = keymap_config.no_gui ? 255 : 0;
+    uint8_t blue = host_keyboard_led_state().scroll_lock ? 255 : 0;
```

Comment:
@HorrorTroll as this is potentially considered a non default keymap, can you sign off on this change please.

Reply:
> @HorrorTroll as this is potentially considered a non default keymap, can you sign off on this change please.

This affects Scroll and Win status LEDs only.
IK75 have an issue, that markings on the PCB are different from the ones on the case.

This is the clip from the correlated issue https://photos.app.goo.gl/WtCdnj2y5uJvJ93b9

You can see that wrong LEDs are activating.

IMHO case markings are more important, because these are the ones you see. PCB is always hidden.

This is indeed a workaround for a specific hardware, but but I don't see any drawbacks.

Reply:
The issue isn't with needing more context on the change.

From <https://docs.qmk.fm/#/breaking_changes_instructions>,

> Edits to User Keymaps A user may submit their keymap to QMK, then some time later open a pull request with further updates, only to find it can’t be merged because it was edited in the qmk/qmk_firmware repository. As not all users are proficient at using Git or GitHub, the user may find themself unable to fix the issue on their own.

Otherwise this change has to go via `develop`.